src/cookbook/maintenance/error-reporting.md

```diff
@@ -23,7 +23,7 @@ How can you determine how often your users experiences bugs?
 Whenever an error occurs, create a report containing the
 error that occurred and the associated stacktrace.
 You can then send the report to an error tracking
-service, such as [Bugsnag][], Fabric, [Rollbar][], or Sentry.
+service, such as [Bugsnag][], Fabric, Firebase Crashlytics, [Rollbar][], or Sentry.
```

Comment:
Add a link?

Reply:
Sure! Should I do it for all services (sentry, rollbar etc...)?

Reply:
Nice to have, not a hard requirement =)

Reply:
Fabric is discontinued by Google perhaps. is it better to remove it?